<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mixeur</class>
 <widget class="QMainWindow" name="Mixeur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pigment_1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pigment_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>110</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cpu_gpu">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>290</y>
      <width>160</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="opengllayout"/>
   </widget>
   <widget class="QLabel" name="label_pourcentage">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>260</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Concentration</string>
    </property>
   </widget>
   <widget class="QLabel" name="RGB">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>480</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RGB :</string>
    </property>
   </widget>
   <widget class="QLabel" name="XYZ">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>400</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>XYZ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>530</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cpu_gpu</sender>
   <signal>clicked()</signal>
   <receiver>cpu_gpu</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
